### PR TITLE
LPS-177435 Web content field values are lost after editing for ca-ES-VALENCIA

### PR DESCRIPTION
**Pull request by ccorreagg**
Followed-up PR from: https://github.com/liferay-headless/liferay-portal/pull/1238

---

This PR fix the bug :https://issues.liferay.com/browse/LPS-177435.

When it is used the Locale.filter method, the method is returned the locale of valencia as ca_ES_valencia and this is cause an issue with the ckeditor of the web content edition fields. So, with this fix we are keeping the locale as it is defined in the portal locales, for valencia language : ca_ES_VALENCIA .

cc/ @Mikellorza

### CODE REVIEW

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java [hunk 1]
```diff
@@ -78,25 +78,23 @@ else if (acceptLanguage.equals("zh-Hant-TW")) {
...
-					if (LocaleUtil.equals(locale, companyAvailableLocale)) {
-						fixedLocales.add(companyAvailableLocale);
-
-						break;
-					}
-				}
-			}
-
-			if (ListUtil.isEmpty(fixedLocales)) {
+			if (ListUtil.isEmpty(locales)) {
```
Comment (4lejandrito) [inline modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java:81]: Shouldn't this if be after the call to `TransformUtil`? The logic is not equivalent to the previous commit. If the transformutil returns null for all elements the exception will not be thrown.

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

File: modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java (hunk 1 above)
Comment (ccorreagg) [reply modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/accept/language/AcceptLanguageImpl.java]: It is not equivalent, but it should not fail. Anyway, as we discussed, I am moving it before the empty validation

**Comment by jalber786** [comment]
ci:stop

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:forward

**Pull request closed without merging**